### PR TITLE
Cleanup behavior and fix some typos

### PR DESCRIPTION
**Pull request by jaagut**
# Summary

Cleanup the behavior and fixing some typos

## Proposed changes

- Removes some unneeded files
- Cleanup some comments
- Typos

## Related issues

Redone from https://github.com/bit-bots/bitbots_behavior/pull/301

## Checklist

- [ ] Run `colcon build`
- [ ] Write documentation
- [ ] Test on your machine
- [ ] Test on the robot
- [ ] Create issues for future work
- [X] Triage this PR and label it

### CODE REVIEW

File: bitbots_behavior/bitbots_blackboard/bitbots_blackboard/capsules/animation_capsule.py [hunk 1]
```diff
@@ -1,9 +1,3 @@
-"""
-AnimationCapsule
```
Comment (Flova) [inline bitbots_behavior/bitbots_blackboard/bitbots_blackboard/capsules/animation_capsule.py:2]: This was deleted, not moved. I think this is still useful information.

File: bitbots_behavior/bitbots_blackboard/bitbots_blackboard/capsules/kick_capsule.py [hunk 2]
```diff
@@ -1,9 +1,3 @@
-"""
-KickCapsule
-^^^^^^^^^^^^^^^^
```
Comment (Flova) [inline bitbots_behavior/bitbots_blackboard/bitbots_blackboard/capsules/kick_capsule.py:3]: Same here

File: bitbots_behavior/bitbots_blackboard/bitbots_blackboard/capsules/misc_capsule.py [hunk 3]
```diff
@@ -1,9 +1,3 @@
-"""
-MiscCapsule
-^^^^^^^^^^^
-
-Capsule for miscellaneous things that don't fit anywhere else.
```
Comment (Flova) [inline bitbots_behavior/bitbots_blackboard/bitbots_blackboard/capsules/misc_capsule.py:5]: And here

File: bitbots_behavior/bitbots_blackboard/bitbots_blackboard/capsules/pathfinding_capsule.py [hunk 4]
```diff
@@ -1,10 +1,3 @@
-"""
-PathfindingCapsule
-^^^^^^^^^^^
-
-Capsule that handles everything related to pathfinding.
```
Comment (Flova) [inline bitbots_behavior/bitbots_blackboard/bitbots_blackboard/capsules/pathfinding_capsule.py:5]: And here

File: bitbots_behavior/bitbots_blackboard/bitbots_blackboard/capsules/team_data_capsule.py [hunk 5]
```diff
@@ -1,7 +1,3 @@
-"""
-TeamDataCapsule
-^^^^^^^^^^^^^^^
```
Comment (Flova) [inline bitbots_behavior/bitbots_blackboard/bitbots_blackboard/capsules/team_data_capsule.py:3]: Okay there was not much here, but we could still add some description to the class

File: bitbots_behavior/bitbots_blackboard/bitbots_blackboard/capsules/world_model_capsule.py [hunk 6]
```diff
@@ -1,10 +1,3 @@
-"""
-WorldModelCapsule
-^^^^^^^^^^^^^^^^^^
-
-Provides information about the world model.
```
Comment (Flova) [inline bitbots_behavior/bitbots_blackboard/bitbots_blackboard/capsules/world_model_capsule.py:5]: Same here

File: bitbots_behavior/bitbots_body_behavior/bitbots_body_behavior/behavior_dsd/actions/go_to.py [hunk 7]
```diff
@@ -50,8 +50,6 @@ def perform(self, reevaluate=False):
 
 
 class GoToAbsolutePosition(AbstractActionElement):
-    blackboard: BodyBlackboard
```
Comment (Flova) [inline bitbots_behavior/bitbots_body_behavior/bitbots_body_behavior/behavior_dsd/actions/go_to.py:53]: Why did you remove this? This way we have typing for the blackboard.

File: bitbots_behavior/bitbots_blackboard/bitbots_blackboard/capsules/animation_capsule.py (hunk 1 above)
Comment (jaagut) [reply bitbots_behavior/bitbots_blackboard/bitbots_blackboard/capsules/animation_capsule.py:2]: Why? It's literally in the file name, does not provide any additional information and we do it nowhere else. To be honest, I think it looks ugly

Comment (Flova) [reply bitbots_behavior/bitbots_blackboard/bitbots_blackboard/capsules/animation_capsule.py:2]: I fat fingered the line. My point focused on the long form description, which is removed without replacement and contains information like the communication with the animation action server and it's purpose.

For other capsules you ported this docstring to the class definition, but for others is was removed.

File: bitbots_behavior/bitbots_blackboard/bitbots_blackboard/capsules/costmap_capsule.py [hunk 8]
```diff
@@ -28,6 +22,8 @@
 
 
 class CostmapCapsule:
+    """Provides information about the cost of different positions and moves."""
```
Comment (Flova) [inline bitbots_behavior/bitbots_blackboard/bitbots_blackboard/capsules/costmap_capsule.py:25]: Here it was moved for example.

File: bitbots_behavior/bitbots_body_behavior/bitbots_body_behavior/behavior_dsd/actions/go_to.py (hunk 7 above)
Comment (jaagut) [reply bitbots_behavior/bitbots_body_behavior/bitbots_body_behavior/behavior_dsd/actions/go_to.py:53]: I'm not sure, but I think it does not help, because only self.bb is used and not bb

Comment (Flova) [reply bitbots_behavior/bitbots_body_behavior/bitbots_body_behavior/behavior_dsd/actions/go_to.py:53]: You can define fields that you access using self. in this scope.

**Review by Flova — APPROVED** [review]
